Frameworks/TensorFlow/Getting-Started-Samples/TensorFlow_HelloWorld/sample.json

```diff
@@ -0,0 +1,21 @@
+{
+  "guid": "111213A0-C930-45B4-820F-02472BABBF34",
+  "name": "Tensorflow Hello World",
+  "categories": ["Toolkit/Intel® AI Analytics Toolkit/TensorFlow"],
+  "description": "This sample shows how to train a TensorFlow and run inference with oneMKL and oneDNN.",
+  "builder": ["cli"],
+  "languages": [{"python":{}}],
+  "os":["linux"],
+  "ciTests": {
+	"linux": [
+	{
+		"env": ["source /opt/intel/oneapi/setvars.sh --force" ],
```

Comment:
Same as for the other sample, why sourcing `setvars.sh`?

Reply:
we want to make sure the environment is setting. thx